app/backend/graphql/project/index.ts

```diff
@@ -1,3 +1,4 @@
-export * from './project'
-export * from './queries'
-export * from './mutations'
+import './queries'
+import './mutations'
+
+export { Project } from './project'
```

Comment:
used where?

Reply:
Sometimes we have to import a pothos object type directly, e.g. we have to import `User` into `Project`.
I would keep it consistent, and always export the object types.